test/test_helper.rb

```diff
@@ -3,6 +3,7 @@
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
+require 'minitest/rails/capybara'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.